src/metric_model_reporter.h

```diff
@@ -73,6 +77,12 @@ struct MetricReporterConfig {
   bool cache_enabled_ = false;
 
   bool is_decoupled_ = false;
+
+ private:
+  // Maps the metric fullname to its lookup key.
+  // All new histograms must update the map.
```

Comment:
Add comment on why you need to have a mapping

Reply:
Updated